test/libsolidity/syntaxTests/errors/basic_usage_parethesized.sol

```diff
@@ -0,0 +1,13 @@
+error E();
+function f() pure {
+    revert((E()));
+}
+function g() pure {
+    bool x;
+    require(x, (E()));
+}
+// ----
+// TypeError 6473: (43-46): Tuple component cannot be empty.
+// TypeError 4423: (42-47): Expected error or string.
+// TypeError 6473: (100-103): Tuple component cannot be empty.
+// TypeError 4423: (99-104): Expected error or string.
```

Comment:
Because of the design of the compiler. I did not want to introduce a new type when you cannot do anything with the object anyway. Or do we want to allow errors to be stored in memory?